src/Moryx.Products.Model/Entities/ProductTypeEntity.cs

```diff
@@ -30,6 +30,15 @@ public class ProductTypeEntity : ModificationTrackedEntityBase
 
         public virtual ProductTypePropertiesEntity CurrentVersion { get; protected internal set; }
 
+        public ProductTypeEntity()
```

Comment:
Can you just add a small explenation why the entities need to be changed, otherwise the PR is good 👍

Reply:
To have their collections (or relations) instantiated by default.
1. This is the way, models in EF Core usually look like (e.g. when created in a DB first approach)
2. Does it prevent null reference exceptions

Reply:
Ok, yeah I just ask myself how it worked before 😅 If we need to do this with all entity classes from now on, it should be in the Migration Guide for Core I think 🤔